foundry.toml

```
@@ -2,12 +2,11 @@
 src = "src"
 test = "test/forge"
 libs = ["lib"]
-evm_version = "cancun"
 via-ir = true
```

```suggestion
via_ir = true
```
(optional) to align with other repos